### ISSUE TITLE
The PBR model does not display correctly when using multiple lights

### ISSUE THREAD

**Issue by cx20** [GrimoireGL/grimoirejs-gltf#41]
It is not displayed correctly when multiple lights are applied to the PBR model.

[Grimoire.js + glTF/MetalRoughSpheres.gltf + Direction light + Point light:](https://cdn.rawgit.com/cx20/gltf-test/f40356df972c5b8181cc4c3db0b0548badff3b0e/examples/grimoiregl/index.html?category=tutorialModels&model=MetalRoughSpheres&scale=0.2&type=glTF)
![image](https://user-images.githubusercontent.com/1833979/27034520-0558bb9a-4fb9-11e7-89eb-6f139fd6a4f7.png)
```html
<light type="directional" rotation="30d,170d,0d"  color="white" intensity="1.0"/> -->
<light type="point" position="0,5,0"  color="white"  distance="50" decay="0.3" intensity="5.0"/>
```

Grimoire.js + glTF/MetalRoughSpheres.gltf + Direction light + Direction light:
![image](https://user-images.githubusercontent.com/1833979/27035446-53e3fb8c-4fbc-11e7-990f-84365c84ecca.png)
```html
<light type="directional" rotation="0d,0d,0d" color="white" intensity="1.0"/>
<light type="directional" rotation="0d,90d,0d" color="white" intensity="1.0"/>
```

**Comment by cx20** [comment]
Probably beyond 1.0 it seems to cause problems.
```glsl
vec3 BRDF(pbr_params params,vec3 li,vec3 lo,vec3 n){
  //return params.diffuseColor/PI + cookTorranceBRDF(params,li,lo,n);
  vec3 result = params.diffuseColor/PI + cookTorranceBRDF(params,li,lo,n);
  vec3 result2 = vec3(0.0);
  result2.x = min(1.0, result.x);
  result2.y = min(1.0, result.y);
  result2.z = min(1.0, result.z);
  return result2;
}
```

![image](https://user-images.githubusercontent.com/1833979/27646187-f6505f0c-5c62-11e7-853a-ea94b1716975.png)

![image](https://user-images.githubusercontent.com/1833979/27646298-3e7a0b7a-5c63-11e7-8648-15a262493c02.png)

**Comment by cx20** [comment]
I confirmed that it was fixed in grimoirejs-forward-shading v1.7.2.
https://github.com/GrimoireGL/grimoirejs-forward-shading/commit/91f9ad0370d5ce3f5de3933675a9cf53fc92220b
